substratools/algo.py

```diff
@@ -415,7 +415,7 @@ def _user_func(args):
             n_fake_samples=args.n_fake_samples,
         )
 
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(fromfile_prefix_chars='@')
```

Comment:
What is the reasoning behind this solution of reading arguments from a file ? Is it because the change is simpler than if you moved to a structured JSON file for communications between the backend (Local or remote) and the tools container ?

Reply:
It's because the cli has a size limite for args. Hence the execution fails if you have more than ±1000 data samples. 
This is happening to Tanguy at the moment on the Lipome project.

Reply:
Yes, I know that issue, what I'm curious about is keep using `argparse` with command line args dumped in a file VS send all the info through a potentially versioned JSON file that we would just load with `json.load()` or using `pydantic`.

Reply:
I'll close this comment since it just asked for clarification. Please feel free to add another comment if you want to challenge the approach itself.